src/esp/gfx/PbrDrawable.h

```diff
@@ -85,6 +86,8 @@ class PbrDrawable : public Drawable {
   Magnum::Resource<Magnum::GL::AbstractShaderProgram, PbrShader> shader_;
   Magnum::Resource<MaterialData, PbrMaterialData> materialData_;
   Magnum::Resource<LightSetup> lightSetup_;
+
+  bool createRenderer_;
```

Comment:
I guess this flag could live in the base Drawable class instead of repeated in several derived Drawable classes.